purchases/src/test/java/com/revenuecat/purchases/PurchasesTest.kt

```diff
@@ -2864,21 +2852,6 @@ class PurchasesTest {
         }
     }
 
-    @Test
-    fun `caches are not cleared if update purchaser info fails`() {
-        mockBackend(PurchasesError(PurchasesErrorCode.StoreProblemError, "Broken"))
-
-        val lock = CountDownLatch(1)
-        purchases.getCustomerInfoWith(onSuccess = {
-            lock.countDown()
-        }, onError = {
-            fail("supposed to be successful")
-        })
-        lock.await(200, TimeUnit.MILLISECONDS)
-        assertThat(lock.count).isZero()
-        verify(exactly = 0) { mockCache.clearCachesForAppUserID(any()) }
```

Comment:
We were not calling `clearCachesForAppUserID` in here at all. I think this was an outdated test?

Reply:
Ok will add it back and rename it or add a comment so it's a bit clearer 👍